release.txt

```diff
@@ -3,5 +3,6 @@ tox
 echo $version
 git tag -a v$version -m v$version
 git push --tags
-python setup.py register sdist
+python -m pip install build
+python -m build
```

Comment:
Would suggest not installing things like this in the users environment and instead document how to do it either here or elsewhere.

```suggestion
# Install `build` if not present with `python -m pip install build` or similar
python -m build
```